core/include/drivers/cbmem_console.h

```diff
@@ -2,26 +2,10 @@
 /*
  * Copyright (c) 2023, Google Limited
  */
-#ifndef CBMEM_CONSOLE_H
-#define CBMEM_CONSOLE_H
-
-#include <types_ext.h>
-#include <drivers/serial.h>
-
-struct cbmem_console {
-	uint32_t size;
-	uint32_t cursor;
-	uint8_t body[0];
-} __packed;
-
-struct cbmem_console_data {
-	paddr_t base;
-	struct cbmem_console *console;
-	struct serial_chip chip;
-	uint32_t size;
-};
+#ifndef DRIVERS_CBMEM_CONSOLE_H
+#define DRIVERS_CBMEM_CONSOLE_H
 
```

Comment:
`#include <stdbool.h>`

Reply:
Done (once you reply to the other comments, then I'll upload a new commit w/ this and any other changes included).